### PR TITLE
Fix routes

### PR DESCRIPTION
**Pull request by Thomas-Nigon**

### CODE REVIEW

File: frontend/src/components/Contexts/navBarContext.jsx [hunk 1]
```diff
@@ -5,10 +5,10 @@ export const NavContext = createContext();
 
 export function NavBarProvider({ children }) {
   const [activeButton, setActiveButton] = useState("home");
-  const [recipeID, setRecipeID] = useState("1");
+  /*   const [recipeID, setRecipeID] = useState("0"); */
```
Comment (Marie-Sail) [inline frontend/src/components/Contexts/navBarContext.jsx:8]: code mort

Comment (Sax593) [inline frontend/src/components/Contexts/navBarContext.jsx:8]: Death code

**Review by NemoOnTheNet — APPROVED** [review]

File: frontend/src/pages/Recipe/Recipe.jsx [hunk 2]
```diff
@@ -1,34 +1,36 @@
...
-import { useEffect, useState, useContext } from "react";
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
 import RecipeInfo from "../../components/RecipeInfo/RecipeInfo";
 import IngredientCard from "../../components/IngredientCard/IngredientCard";
 import AddComment from "../../components/AddComment/AddComment";
 import RecipeStep from "../../components/RecipeStep/RecipeStep";
 import UstensiltCard from "../../components/UtensilCard/UtensilCard";
-import { NavContext } from "../../components/Contexts/navBarContext";
+/* import { NavContext } from "../../components/Contexts/navBarContext"; */
```
Comment (Sax593) [inline frontend/src/pages/Recipe/Recipe.jsx:9]: Death code

File: frontend/src/pages/Recipe/Recipe.jsx [hunk 3]
```diff
@@ -1,34 +1,36 @@
...
 import CommentCard from "../../components/CommentCard/CommentCard";
 
 function Recipe() {
+  const [recipe, setRecipe] = useState();
+  const { recipeID } = useParams();
+
```
Comment (Sax593) [inline frontend/src/pages/Recipe/Recipe.jsx:15]: unnecessary space

**Review by Sax593 — CHANGES_REQUESTED** [review]

**Review by Marie-Sail — APPROVED** [review]

**Review by Sax593 — APPROVED** [review]

**Pull request merged**